kurtosis-core-rpc-api/api_container_service.proto

```diff
@@ -339,3 +362,19 @@ message ExecuteBulkCommandsArgs {
   // Commands, serialized as JSON
   string serialized_commands = 1;
 }
+
+// ==============================================================================================
+//                                          Get Services
+// ==============================================================================================
+message GetServicesResponse {
+  // List of services' id
+  repeated string services_id = 1;
```

Comment:
1) same rules from above, so would name this `service_ids`
2) I'd make this a "set" (Protobuf doesn't have sets either, so a `map[string]bool`)
3) comment needs updating